tools/perf/perf_run.py

```diff
@@ -527,7 +528,6 @@ def recordStats(backend, timings, precision, batch_size=1, compile_time_s=None):
     )
     args = arg_parser.parse_args()
 
-    cudnn.benchmark = True
```

Comment:
@narendasan - this line causes a segfault at inference time, when we compile the Docker container with CUDNN 8.8, and Torch 2.1.0 uses the CUDNN 8.9 Python distributions. When removed, inference works as expected.

Do you think it would be necessary/important to upgrade the build stack to CUDNN 8.9 for the upcoming release?